### PR TITLE
Jaytee ramen

### PR DESCRIPTION
**Pull request by amriikk**
init

### CODE REVIEW

**Comment by amriikk** [comment]
update css

File: jaytee/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&display=swap" rel="stylesheet">
+    <title>Ramen Recipe</title>
+</head>
+<body>
+    <h1>Ramen Recipe</h1>
+    <div class="ingredients-list">
+        <h2 onclick=listIngredients()>Ingredients</h2>
+        <ul>
+            {Ingredients.map(name => <li key={name}> {name} </li>)}
```
Comment (sugarfig) [inline jaytee/index.html]: prints out to screen. not sure if it was intended to do that.

File: jaytee/js/script.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const ingredients = ['1 tablespoon sesame oil', '3 teaspoons grated ginger', '4 teaspoons grated garlic', '4 cups broth', '4 cups water', '1 ounce dried shiitake mushrooms', '2 packages instant ramen (noodles only!)', '1/2 cup chopped scallions or chives', '2 cup chopped kale', '1 cups shredded carrots'];
```
Comment (sugarfig) [inline jaytee/js/script.js:1]: ingredients never get printed.

**Review by sugarfig — CHANGES_REQUESTED** [review]

**Comment by amriikk** [comment]
updated the broken code in html index

**Review by sugarfig — APPROVED** [review]
good job at fixing changes !

**Pull request merged**